### PR TITLE
plan: let explain show count

### PR DESCRIPTION
**Pull request by hanfei1991**
make explain result more clearly.

### CODE REVIEW

**Comment by lamxTyler** [comment]
LGTM

File: plan/planbuilder.go [hunk 1]
```diff
@@ -906,11 +906,12 @@ func (b *planBuilder) buildExplain(explain *ast.ExplainStmt) Plan {
 	setParents4FinalPlan(targetPlan.(PhysicalPlan))
 	p := &Explain{StmtPlan: targetPlan}
 	if UseDAGPlanBuilder(b.ctx) {
-		retFields := []string{"id", "parents", "task", "operator info"}
+		retFields := []string{"id", "parents", "children", "task", "operator info"}
 		schema := expression.NewSchema(make([]*expression.Column, 0, len(retFields))...)
 		for _, fieldName := range retFields {
 			schema.Append(buildColumn("", fieldName, mysql.TypeString, mysql.MaxBlobWidth))
 		}
+		schema.Append(buildColumn("", "count", mysql.TypeDouble, mysql.MaxBlobWidth))
```
Comment (zz-jason) [inline plan/planbuilder.go]: s/mysql.MaxBlobWidth/mysql.MaxRealWidth/

File: plan/planbuilder.go [hunk 2]
```diff
@@ -906,11 +906,12 @@ func (b *planBuilder) buildExplain(explain *ast.ExplainStmt) Plan {
 	setParents4FinalPlan(targetPlan.(PhysicalPlan))
 	p := &Explain{StmtPlan: targetPlan}
 	if UseDAGPlanBuilder(b.ctx) {
-		retFields := []string{"id", "parents", "task", "operator info"}
+		retFields := []string{"id", "parents", "children", "task", "operator info"}
```
Comment (zz-jason) [inline plan/planbuilder.go]: "parents", "children" no need to appears together

Comment (hanfei1991) [inline plan/planbuilder.go]: Inferring children by parent is very very very very painful.

**Pull request merged**